pkg/minikube/problem/err_map.go

```diff
@@ -36,6 +36,11 @@ var vmProblems = map[string]match{
 		URL:    "https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver",
 		Issues: []int{1926, 4206},
 	},
+	"OLD_HYPERKIT_VERSION": {
+		Regexp: re(`Too old hyperkit version found`),
+		Advice: "too old hyperkit version. Jump forward to new valid hyperkit version as a workaround.",
+		Issues: []int{4658},
```

Comment:
Add URL to documentation on how to upgrade.

Reply:
Add this doc: https://github.com/kubernetes/minikube/blob/master/docs/drivers.md

Reply:
Done!

Reply:
done